grafana-plugin/src/models/loader/loader.ts

```diff
@@ -13,13 +13,15 @@ class LoaderStoreClass {
   }
 
   @action.bound
-  setLoadingAction(actionKey: string, isLoading: boolean) {
-    this.items[actionKey] = isLoading;
+  setLoadingAction(actionKey: string | string[], isLoading: boolean) {
+    if (typeof actionKey === 'string') {
```

Comment:
`Array.isArray(actionKey)`

Reply:
Yes, it will work the same but could you explain the benefit, pls?

Reply:
my proposal sounds like "if actionKey is an array then iterate, otherwise use it as is", yours sounds like "if it's a string then use  it as is otherwise iterate", I think my approach is more safe

Reply:
We're basically deciding here between:
1. My version:
```
if (conditionForSimplerScenario) {
  // cover simple scenario
} else {
  // cover more complex scenario
}
```
2. Your proposal
```
if (conditionForMoreComplexScenario) {
  // cover more complex scenario
} else {
  // cover simple scenario
}
```

I honestly prefer the first one in general, it's similar when you return value from React component. Sometimes if for example you have no data you return `null` (simple scenario). It's more readable to have it before you write some complex JSX (more complex scenario). But anyway it's so minor that let's not spend time discussing it now, I'll just use your suggestion :)